src/button/Button.tsx

```diff
@@ -73,16 +84,9 @@ function Button({
       onBlur={onBlur}
       disabled={isButtonDisabled}
       aria-label={ariaLabel}>
-      {shouldDisplaySpinner ? (
-        <Spinner
-          customClassName={classNames("button__spinner", {
-            [`${customClassName}__spinner`]: customClassName
-          })}
-          aria-label={"Button spinner visible. Button inactivated."}
-        />
-      ) : (
-        children
-      )}
+      <div className={childrenClassName}>{children}</div>
```

Comment:
Adding a wrapper div to the children might require many changes in the projects that use Button component. I wonder if there is a way to hide the children of `button` tag that is not the spinner, `not` utility of CSS may help:

```
.button--is-pending {
   & > *:not(.button__spinner)
}
```

Reply:
You are right. But when we add this CSS, `.button__spinner` isn't working correctly. As we know, I guess it has not specific enough. However, it's a little bit hacky but when we use like following, looks working. 
```
.button--is-pending {
  color: var(--button-bg);
}
```

Reply:
This may lead to strange bugs, we should find a better solution.